modules/social_features/social_event/modules/social_event_max_enroll/social_event_max_enroll.module

```diff
@@ -106,6 +111,17 @@ function social_event_max_enroll_form_alter(&$form, FormStateInterface $form_sta
       $event_max_enroll_service = \Drupal::service('social_event_max_enroll.service');
       $node = \Drupal::routeMatch()->getParameter('node');
 
+      // If there is node ID instead of an object then load node with NID.
+      if (
+        !($node instanceof NodeInterface) &&
+        !is_null($node)
```

Comment:
This statement in theory is still allowing other instances, is_numeric check would be more fitting.

Reply:
Done